package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "stream-chat-docusaurus-cli",
+  "name": "stream-docusaurus-cli",
```

Comment:
I do understand that this is correct, but I'm not ready to propagate this change to the entire FE Chat SDK department, as dozens of engineers will have to get the memo, update their workflows, etc. It will also break existing automation. Let's rename it when we gather more muscle and a clear owner of the docusaurus.